packages/pluggableWidgets/activity-indicator-native/src/assets/ActivityIndicatorPrimary.ts

```diff
@@ -0,0 +1,18 @@
+export const ActivityIndicatorPrimarySVG = () =>
```

Comment:
Is there a specific reason to go for a function here? I think you can just assign the string literal to the constant here.